### PR TITLE
Unknown provider is error [BT-199]

### PR DESCRIPTION
**Pull request by mcovarr**
~Just a draft for now, unit tests are passing but coverage is lacking and integration tests are red.~ Review away!

### CODE REVIEW

**Review by kshakir — COMMENTED** [review]
Repeating what the bot said: some of the new errors do not seem to have unit tests.

**Review by kshakir — APPROVED** [review]

File: test/martha/martha_v3.test.js [hunk 1]
```diff
@@ -965,24 +946,25 @@ function determineDrsTypeTestWrapper(testUrl) {
...
 test.serial('martha_v3 determineDrsType should parse drs:// Data Object uri with query part', (t) => {
     t.is(
-        determineDrsTypeTestWrapper('drs://fo.o/bar?version=1&bananas=yummy'),
-        'https://fo.o/ga4gh/dos/v1/dataobjects/bar?version=1&bananas=yummy'
+        determineDrsTypeTestWrapper(`drs://${bdc}/bar?version=1&bananas=yummy`),
+        `https://${bdc}/ga4gh/drs/v1/objects/bar?version=1&bananas=yummy`
     );
 });
 
 test.serial('martha_v3 determineDrsType should parse drs:// Data Object uri when host includes a port number', (t) => {
```
Comment (breilly2) [inline test/martha/martha_v3.test.js:963]: Looks like this is actually illegal, according to the spec (https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.1.0/docs/#_compact_identifier_based_drs_uris):
> The colon character is not allowed in a Hostname-based DRS URI, making it easy to tell them apart.

**Review by breilly2 — APPROVED** [review]
👍  Looks good! Glad to finally be doing this!

File: test/martha/martha_v3.test.js (hunk 1 above)
Comment (mcovarr) [reply test/martha/martha_v3.test.js:963]: Uhh not sure what to do here as I suspect this test was added to accommodate a real world example... the test treats the number after the colon as a port but the docs are saying that's actually an 'accession'.  🙁

Comment (mcovarr) [reply test/martha/martha_v3.test.js:963]: Written up as BT-345.

**Pull request merged**